security/permissions/src/main/resources/application.properties

```diff
@@ -28,3 +28,16 @@ quarkus.http.auth.policy.role-policy2.permissions.admin=read:minimal,read:all,cr
 quarkus.http.auth.permission.roles2.paths=/default-permission-allowed/custom-permission/*,/custom-annotation-permission-allowed/custom-permission/*
 quarkus.http.auth.policy.role-policy2.permission-class=io.quarkus.ts.openshift.security.basic.permissions.resources.helpers.CustomPermission
 quarkus.http.auth.permission.roles2.policy=role-policy2
+
+quarkus.http.auth.permission.roles3.shared=true
```

Comment:
Is this needed? The `roles3` permission is used only on path. The other permissions are on different paths.

Reply:
Not needed